content/pages/links.md

```diff
@@ -0,0 +1,23 @@
+# Link to External Resources
+
+The members of Project Pythia approve of these resources on Python specific topics.
```

Comment:
Approve seems a bit strong. How about "Project Pythia find the links below to be useful resources on a variety of Python topics." ?

Reply:
I think that would be better, avoiding appearance of some ill-defined "approval" process.

But otherwise all looks good.

Reply:
Good idea! I was uncertain on the best language to use.